airbyte-integrations/connectors/source-blob/source_blob/spec.json

```diff
@@ -0,0 +1,108 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Blob Source Spec",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["dataset_name", "provider", "format", "path"],
+    "properties": {
+      "dataset_name": {
+        "type": "string",
+        "description": "Name of the final table (should include only letters, numbers, dash and underscores. Note that this may be altered according to destination.)"
+      },
+      "provider": {
+        "type": "object",
+        "description": "Storage Provider or Location of the file(s) to be replicated.",
+        "oneOf": [
+          {
+            "title": "S3: Amazon Web Services",
+            "required": ["storage", "bucket"],
+            "properties": {
+              "storage": {
+                "type": "string",
+                "enum": ["S3"],
+                "default": "S3"
+              },
+              "bucket": {
+                "type": "string",
+                "description": "Name of the S3 bucket where the file(s) exist."
+              },
+              "aws_access_key_id": {
+                "type": "string",
+                "description": "In order to access private Buckets stored on AWS S3, this connector would need credentials with the proper permissions. If accessing publicly available data, this field is not necessary."
+              },
+              "aws_secret_access_key": {
+                "type": "string",
+                "description": "In order to access private Buckets stored on AWS S3, this connector would need credentials with the proper permissions. If accessing publicly available data, this field is not necessary.",
+                "airbyte_secret": true
+              },
+              "path_prefix": {
+                "type": "string",
+                "description": "By providing a path-like prefix (e.g. \"myFolder/thisTable/\") to the desired blobs, we can optimise finding these in S3. This is recommended if your bucket contains many folders/files."
+              }
+            }
+          }
+        ]
+      },
+      "format": {
+        "type": "object",
+        "description": "File Format of the file to be replicated (Warning: some format may be experimental, please refer to docs).",
+        "oneOf": [
+          {
+            "title": "csv",
+            "description": "see details <a href=\"https://arrow.apache.org/docs/python/generated/pyarrow.csv.read_csv.html#pyarrow-csv-read-csv\" target=\"_blank\">here</a>",
+            "required": ["filetype"],
+            "properties": {
+              "filetype": {
+                "type": "string",
+                "enum": ["csv"],
+                "default": "csv"
+              },
+              "delimiter": {
+                "type": "string",
+                "description": "leave blank to default to , (comma)"
```

Comment:
Should be the default value be explicitly specified here? I could be wrong, but I think when left blank, the `delimiter` will just return empty string. So the `delimiter=self._format.get("delimiter", ',')` in `filereader.py` will get `""` instead of `","`.

Reply:
addressed